config/secrets.yml

```diff
@@ -0,0 +1,35 @@
+# Be sure to restart your server when you modify this file.
+
+# Your secret key is used for verifying the integrity of signed cookies.
+# If you change this key, all old signed cookies will become invalid!
+
+# Make sure the secret is at least 30 characters and all random,
+# no regular words or you'll be exposed to dictionary attacks.
+# You can use `rake secret` to generate a secure secret key.
+
+# Make sure the secrets in this file are kept private
+# if you're sharing your code publicly.
+
+defaults: &defaults
+  # You can generate secret keys with 'rake secret'
+
+  # For rails cookies
+  secret_key_base: '9edf3727559052d8c92c141b559886c09db26a2e3fc640cb53f1f773ee8415972069b5496202c85340f8fd455ae37e919a1b60c39b6b15ed72f6b8795bdda60d'
+
+  # For devise login tokens
+  devise_secret_key: '775ce72c7fe6b8ea2aaa62d8e23be8a7aa842741cba07f66f11308acd921592b755ed6ab42e58ea2c404272e8e91fea9b29c39c13d5c55f354dbb5db66e57060'
+
+  # Your errbit API key
+  # errbit_key: '12345'
+
+development:
+  <<: *defaults
+
+test:
+  <<: *defaults
+
+# Do not keep production secrets in the repository,
+# instead read values from the environment.
+production:
+  secret_key_base: <%= ENV["SECRET_KEY_BASE"] %>
+  devise_secret_key: <%= ENV["SECRET_KEY_BASE"] %>
```

Comment:
Once leaked both of them will have same consequence, in case of devise hacker will be able to send request to reset password, actual mail doesn't need to be sent to user email, he just needs the reset link (hence gain control of account) and in case of rails he will be able to forge any user session(hence again gain control of account). 

But yeah.. lets use two different ones.